conductor/resources/submitter.ui

```diff
@@ -527,48 +562,78 @@
             </widget>
            </item>
            <item>
-            <widget class="QCheckBox" name="ui_preemptible_chkbx">
-             <property name="layoutDirection">
-              <enum>Qt::RightToLeft</enum>
-             </property>
-             <property name="text">
-              <string>Preemptible</string>
-             </property>
-             <property name="isFileUserPref" stdset="0">
-              <bool>true</bool>
-             </property>
-            </widget>
-           </item>
-           <item>
-            <widget class="QWidget" name="ui_resource_wgt" native="true">
+            <widget class="QWidget" name="ui_preemptible_wgt" native="true">
              <property name="toolTip">
-              <string>[optional] Specify a resource string to associate the Conductor job with</string>
+              <string>A preemptible instance can be up to 50% less expensive, but could be terminated before it completes a task.
+Preemption rate ~10%.  Recommended best use when renders are short, or not against a mission critical deadline.</string>
              </property>
-             <layout class="QHBoxLayout" name="horizontalLayout_10">
-              <property name="spacing">
-               <number>20</number>
-              </property>
+             <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
-               <widget class="QLabel" name="label_6">
+               <widget class="QCheckBox" name="ui_preemptible_chkbx">
+                <property name="layoutDirection">
+                 <enum>Qt::RightToLeft</enum>
+                </property>
                 <property name="text">
-                 <string>Project</string>
+                 <string/>
+                </property>
+                <property name="isFileUserPref" stdset="0">
+                 <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
-               <widget class="QComboBox" name="ui_project_cmbx">
-                <property name="sizePolicy">
```

Comment:
This section needs to be added back in, it looks like

Reply:
Looks like it was moved to the top of the screen.  Looks like it's probably expected.

Reply:
It jumped up to line 191

Reply:
I moved it to the top of the UI

Reply:
@WMDConductor do you want to weigh in on the new placement of the Project box (and reorganization of the instance information)?

Orig vs new:
![submiiter-new_v_orig](https://user-images.githubusercontent.com/10409070/34273768-99b67cc8-e64a-11e7-9e02-61ca025c4ca3.gif)

Reply:
This gets my thumbs up - picking your project first and then setting your parameters / instance type is a reasonable "top-down" flow.